src/ex1/style.css

```diff
@@ -0,0 +1,72 @@
+.container {
+    padding: 15px;
+    border-radius: 7px;
+    max-width: 50%;
+    margin: auto;
+    background-color: aliceblue;
+    width: 500px;
+}
+
+body {
+    background-color: darkslategray;
+}
+
+h1 {
+    text-align: center;
+    margin-left: auto;
+    padding: auto;
+}
+
+img {
+    display: block;
+    margin: auto;
+    width: 50%;
+}
+
+
+
+.removeTask-btn {
+    background-color: azure;
+    cursor: pointer;
+    float: right;
+    height: auto;
+    width: auto;
+}
+
+
+
+.taskDone-btn {
+    background-color: azure;
+    float: right;
+    height: 2p0;
```

Comment:
This value isn't valid
![image](https://user-images.githubusercontent.com/36815567/169703038-f0c354b5-b840-465f-81c4-79e124f9f8a9.png)